_templates/playground/new/prompt.js

```diff
@@ -0,0 +1,32 @@
+// see types of prompts:
+// https://github.com/enquirer/enquirer/tree/master/examples
+//
+module.exports = [
+  {
+    type: 'input',
+    name: 'name',
+    message: 'Which component is this playground for?',
+    initial: 'ion-button',
+  },
+  {
+    type: 'input',
```

Comment:
If the inputs collide with an existing directory then skipping it still yields a message that indicates new files were created:


```
✔ Which component is this playground for? · ion-item-sliding
✔ What should the playground path be? · basic
✔ Select the Ionic Framework version for the playground · 7
✔ Do you want CSS scaffolded? · No / Yes

Loaded templates: _templates
✔      exists: static/usage/v7/item-sliding/basic/angular.md. Overwrite? (y/N):  (y/N) · false

     skipped: static/usage/v7/item-sliding/basic/angular.md
✔      exists: static/usage/v7/item-sliding/basic/demo.html. Overwrite? (y/N):  (y/N) · false

     skipped: static/usage/v7/item-sliding/basic/demo.html
✔      exists: static/usage/v7/item-sliding/basic/index.md. Overwrite? (y/N):  (y/N) · false

     skipped: static/usage/v7/item-sliding/basic/index.md
✔      exists: static/usage/v7/item-sliding/basic/javascript.md. Overwrite? (y/N):  (y/N) · false

     skipped: static/usage/v7/item-sliding/basic/javascript.md
✔      exists: static/usage/v7/item-sliding/basic/react.md. Overwrite? (y/N):  (y/N) · false

✔      exists: static/usage/v7/item-sliding/basic/vue.md. Overwrite? (y/N):  (y/N) · false     skipped: static/usage/v7/item-sliding/basic/vue.md
       shell: echo `
Generation complete! To use this new component in a docs markdown file, include
the following:

## Basic

import Basic from '@site/static/usage/v7/item-sliding/basic/index.md';

<Basic />
`
```

Reply:
Nit: The options show "y/N" but the default value is "false" which is a bit confusing

Reply:
Also is there a message if this script fails?

Reply:
I guess we should just remove the markdown stuff from getting displayed here and try to rework it later? The way it works right now, it doesn't actually have a way to know if generation succeeded or not. (See my other comment for context.)

Reply:
Ok no big deal if you can't change the `false` part 👍